### DISCUSSION TITLE
Correct way to use the cli with the new features?

### DISCUSSION THREAD

**Discussion by odelma** [bimmerconnected/bimmer_connected#237] [Q&A] [answered]
I am updating some information to Domoticz simply via the status command in the cli and reading the output. Now I'm not sure what I am doing wrong but it seems that the new information that was added is not in the actual ouput that my script is reading, however I can see it on DEBUG line, e.g.:
```
DEBUG:bimmer_connected.state:{'STATUS': {'vin': 'some_vin', 'mileage': 81816,
,...,
,'socmax': 4.819999}}
DEBUG:bimmer_connected.state:received new data from connected drive
```
This information is included in the actual readable output that starts after:
```
Found 1 vehicles: 330e iPerformance
VIN: some_vin
Mileage: 81816
Vehicle properties:
{
    "vin": "some_vin",
    "model": "330e iPerformance",
    "bodytype": "F30",
    "driveTrain": "PHEV",
    "..."
}
```
What should I do differently to be able to get the new information into readable format? 😄

**Comment by gerard33** [comment, marked as answer]
I have anomized the VIN in your code as that is not something you should share on internet (assuming you hadn't changed it).

Which new information you are looking for? If it's the `socmax` that is not yet part of the output.

**Reply by odelma** [reply]
Yes, I had changed the VIN a bit :)

But how come socmax (and other, e.g. electrict drive percentage info) is shown on the debug line output - shouldn’t it be possible to have it also in the standard output?

**Reply by gerard33** [reply]
Yeah it's on our list -> https://github.com/bimmerconnected/bimmer_connected/issues/158
Time is the limit here 😎

Please add other data you would like to see as well to that issue.

**Reply by odelma** [reply]
Ah, ok - great. I somehow thought from rikroe’s comment it was about HA integration 😃 

I would add all the info that is available and currently missing from the status output. Is there some reason for leaving something out 🤔
